src/webgpu/api/operation/render_pass/storeOp.spec.ts

```diff
@@ -0,0 +1,23 @@
+export const description = `API Operation Tests for RenderPass StoreOp.
+
+  Test Coverage Needed:
+
+  - Test that a render pass has correct output for combinations of:
+    - All color attachments from '0' to 'MAX_COLOR_ATTACHMENTS' with combinations of:
+      - storeOp set to {'clear', 'store', 'undefined}
+      - All color renderable formats
+      - mip level set to {'0', mip > '0'}
+      - array layer set to {'1', layer > '1'}
```

Comment:
nit: array layer set to 0 and >1

Reply:
>  array layer set to 0 and >1 for 2D textures

Do you mean array layer set to 0 and **>0** for 2D textures? where array layer means `GPUTextureView.baseArrayLayer = {0, > 0}` and `GPUTextureView.arrayLayerCount = 1`

Reply:
This was for the GPUTextureView baseArrayLyer. Of course if it is >0, then the texture needs to have depth > 1 (arrayLayerCount was folded in depth recently)

Reply:
Okay, done.